dll/win32/advapi32/reg/reg.c

```diff
@@ -39,30 +39,41 @@ static BOOLEAN DllInitialized = FALSE; /* HACK */
 
 /* PROTOTYPES ***************************************************************/
 
-static NTSTATUS MapDefaultKey (PHANDLE ParentKey, HKEY Key);
+static NTSTATUS MapDefaultKey(PHANDLE ParentKey, HKEY Key);
 static VOID CloseDefaultKeys(VOID);
-#define ClosePredefKey(Handle)                                                 \
-    if ((ULONG_PTR)Handle & 0x1) {                                             \
-        NtClose(Handle);                                                       \
-    }
-#define IsPredefKey(HKey)                                                      \
-    (((ULONG_PTR)(HKey) & 0xF0000000) == 0x80000000)
-#define GetPredefKeyIndex(HKey)                                                \
-    ((ULONG_PTR)(HKey) & 0x0FFFFFFF)
-
 static NTSTATUS OpenClassesRootKey(PHANDLE KeyHandle);
-static NTSTATUS OpenLocalMachineKey (PHANDLE KeyHandle);
-static NTSTATUS OpenUsersKey (PHANDLE KeyHandle);
+static NTSTATUS OpenLocalMachineKey(PHANDLE KeyHandle);
+static NTSTATUS OpenUsersKey(PHANDLE KeyHandle);
 static NTSTATUS OpenCurrentConfigKey(PHANDLE KeyHandle);
 
+/* INLINE FUNCTIONS *********************************************************/
```

Comment:
Not sure it's useful to have this distinction between "inline functions" vs. "functions" in these decorative titles.

Reply:
Maybe not, I'll revert.